test/drenv/envfile.py

```diff
@@ -2,9 +2,30 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import copy
+import platform
 
 import yaml
 
+VM = "drenv-vm"
+CONTAINER = "drenv-container"
+SHARED_NETWORK = "drenv-network-shared"
+
+
+def platform_defaults():
+    # Let minikube defaults and errors apply if unknown platforms
+    d = {VM: "", CONTAINER: "", SHARED_NETWORK: ""}
+
+    if platform.platform() == "linux":
+        d[VM] = "kvm2"
+        d[CONTAINER] = "podman"
+        d[SHARED_NETWORK] = "default"
+    elif platform.platform() == "darwin":
+        d[VM] = "hyperkit"
+        d[CONTAINER] = "podman"
+        d[SHARED_NETWORK] = ""
```

Comment:
Checking coverage:

    make test
    make coverage-html

On Linux, shows that this code is not tested - this is expected for the "darwin" branch
but not for the "linux" branch. The reason is:

```
>>> platform.platform()
'Linux-6.1.10-200.fc37.x86_64-x86_64-with-glibc2.36'
```

I think we should use:

```
>>> platform.system().lower()
'linux'
```

But I don't have darwin system around now to test.